src/Symfony/Component/DependencyInjection/Loader/schema/services.schema.json

```
@@ -203,7 +203,7 @@
       ]
     },
     "serviceShortSyntax": {
-      "$comment": "Short syntax: service defined as arguments array. YAML example: my.service: ['@dependency', 'literal', 123]",
+      "$comment": "Short syntax: service defined as arguments array. YAML example: App\\MyService: ['@dependency', 'literal', 123]",
```

```suggestion
      "$comment": "Short syntax: service defined as arguments array. YAML example: App\\SomeService: ['@dependency', 'literal', 123]",
```

I'm not a fan of using a `My` prefix for examples (I know I used to do that as well ;)).